hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/DatanodeStateMachine.java

```diff
@@ -281,8 +281,7 @@ public DatanodeStateMachine(DatanodeDetails datanodeDetails,
   private int getEndPointTaskThreadPoolSize() {
     // TODO(runzhiwang): current only support one recon, if support multiple
     //  recon in future reconServerCount should be the real number of recon
-    int reconServerCount = 1;
-    int totalServerCount = reconServerCount;
```

Comment:
The comment mentions `reconServerCount`.  If you remove that variable, please adjust the comment accordingly.